### DISCUSSION TITLE
qt teamtalk. The last edited lable is only on baysick information tab

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#2360] [Ideas]
### Description

this last edited lable is really useful for people to see when their account was edited at last, but this lable is not related to any tab, and it should be mooved out of the tabs, and it should be placed after cansle button or after close button in view mode, and befor the tab controle, so it will be accessible when any tab is selected. @CoBC will you please do this? I think it is a nice suggestion, thanks!

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by CoBC** [comment]
IMO this info is "basic info", don't see why we should move it out of tab

**Comment by amirmahdifard** [comment]
because then it will be accessible when any tab is selected for quick access, but ok, you can decide, it was just a simple suggestion, sorry
